src/test/java/io/xpring/xrpl/FakeXrpProtobufs.java

```diff
@@ -423,6 +424,82 @@
           .addTransactions(getTransactionResponseCheckCash)
           .build();
 
+  // AccountSet fake primitive test values
+  public static Integer testClearFlag = 5;
+  public static String testDomain = "testdomain";
+
+  public static ByteString testEmailHash;
+  static {
+    try {
+      testEmailHash = ByteString.copyFrom("emailhash", "Utf8");
+    } catch (UnsupportedEncodingException exception) {
+      exception.printStackTrace();
+    }
+  }
+
+  public static ByteString testMessageKey;
+  static {
+    try {
+      testMessageKey = ByteString.copyFrom("messagekey", "Utf8");
+    } catch (UnsupportedEncodingException exception) {
+      exception.printStackTrace();
+    }
+  }
+
+  public static Integer testSetFlag = 4;
+  public static Integer testTransferRate = 1234567890;
+  public static Integer testTickSize = 7;
```

Comment:
let me know if you think this grouping is reasonable - i also wasn't sure if we wanted to group the `Common` protobufs together. lmk